### PR TITLE
Make Postgres surfacer writeChan configurable

### PR DESCRIPTION
**Pull request by benner**
In some cases it is easy to hit 1000 limit. This makes this limit configurable.

Also increasing to 10k which is consitent to Prometheus surfacer.

### CODE REVIEW

File: surfacers/postgres/proto/config.proto [hunk 1]
```diff
@@ -7,4 +7,5 @@ option go_package = "github.com/google/cloudprober/surfacers/postgres/proto";
 message SurfacerConf {
   required string connection_string = 1;
   required string metrics_table_name = 2;
+  optional int64 max_concurrent_inserts = 3 [default = 1000];
```
Comment (manugarg) [inline surfacers/postgres/proto/config.proto:10]: Can you please name it metrics_buffer_size and set default to 10000. Impact of larger buffer size is pretty small. These changes will make it consistent with prometheus surfacer:
https://github.com/google/cloudprober/blob/037977dbc1716b367529d276c471571d39690dfc/surfacers/prometheus/proto/config.proto#L13

**Review by manugarg — COMMENTED** [review]
Thanks a lot for the PR.

File: surfacers/postgres/proto/config.proto (hunk 1 above)
Comment (benner) [reply surfacers/postgres/proto/config.proto]: Done

**Comment by manugarg** [comment]
Thanks, I've started the process of importing it as per https://github.com/google/cloudprober/blob/master/CONTRIBUTING.md#source-of-truth-sot-and-commit-process.

**Comment by manugarg** [comment]
Committed (https://github.com/google/cloudprober/commit/3d5080b5dd0ee6a23395e6cf42a24c3e10557c2d) through #509.

Thank you once again for the contribution, @benner. Much appreciated.

**Pull request closed without merging**